README.md

```diff
@@ -2,92 +2,135 @@
+import ratings from "@mtucourses/rate-my-professors";
 
-  console.log(teachers);
+(async () => {
+  // Search for a school
+  const schools = await ratings.searchSchool("arizona state university");
   /*
-    [
-      {
-        firstName: 'Ching-Kuang',
-        id: 'VGVhY2hlci0yMjkxNjI=',
-        lastName: 'Shene',
-        school: {
-          id: 'U2Nob29sLTYwMg==',
-          name: 'Michigan Technological University'
-        }
-      }
-    ] 
+  [
+    ...response truncated for brevity
+    {
+      city: '',
+      id: 'U2Nob29sLTEzNjQ3',
+      name: 'Arizona State University',
+      state: ''
+    },
+    {
+      city: 'Tucson',
+      id: 'U2Nob29sLTE1NzIz',
+      name: 'Arizona State University',
+      state: 'AZ'
+    },
+    ...response truncated for brevity
+  ]
   */
 
-  const teacher = await ratings.getTeacher('VGVhY2hlci0yMjkxNjI=');
-
-  console.log(teacher);
+  // Search for a teacher at a specific school
+  const teachers = await ratings.searchTeacher(
+    "steven osburn",
+    "U2Nob29sLTEzNjQ3"
+  );
   /*
+  [
     {
-      avgDifficulty: 4.4,
-      avgRating: 3.3,
-      numRatings: 28,
-      department: 'Computer Science',
-      firstName: 'Ching-Kuang',
-      id: 'VGVhY2hlci0yMjkxNjI=',
-      lastName: 'Shene',
-      school: {
-        city: 'Houghton',
-        id: 'U2Nob29sLTYwMg==',
-        name: 'Michigan Technological University',
-        state: 'MI'
-      },
-      legacyId: 1234567
+      firstName: 'Steven',
+      id: 'VGVhY2hlci0yMzAwOTAw',
+      lastName: 'Osburn',
+      school: { id: 'U2Nob29sLTEzNjQ3', name: 'Arizona State University' }
     }
+  ]
+  */
+
+  // Get detailed information about a teacher
+  const teacher = await ratings.getTeacher("VGVhY2hlci0yMzAwOTAw");
+  /*
+  {
+    avgDifficulty: 2.5,
+    avgRating: 4.3,
+    department: 'Engineering',
+    firstName: 'Steven',
+    id: 'VGVhY2hlci0yMzAwOTAw',
+    lastName: 'Osburn',
+    legacyId: 2300900,
+    numRatings: 79,
+    school: {
+      city: '',
+      id: 'U2Nob29sLTEzNjQ3',
+      name: 'Arizona State University',
+      state: ''
+    },
+    wouldTakeAgainPercent: 88.8889
+  }
   */
 })();
 ```
 
-## 🧰  Development
+# API
+
+### `searchSchool(searchQuery: string): Promise<School[]>`
+
+Searches for a school by its name.
+
+### `searchTeacher(searchQuery: string): Promise<Teacher[]>`
+
+Searches for a teacher by their name or school abbreviation.
+
+### `getTeacher(teacherId: string): Promise<TeacherDetails>`
+
+Retrieves detailed information about a teacher by their ID.
+
+# Development
+
+## Setup
 
 ```bash
-# First:
-# install dependencies
-yarn install
+# Install dependencies
+npm install
+```
 
-# then:
-# build in watch mode
-yarn build:watch
+## Build
 
-# and you can:
+```bash
+# Build in watch mode
+npm run build:watch
+```
+
+## Testing
 
-# run tests
-yarn test
+```bash
+# Run tests
+npm test
 
-# run tests in watch mode
-yarn test:watch
```

Comment:
very minor but I like the old format here haha

Reply:
Didn't you mention that `yarn` is not recommended, so I just used npm for all of them.

Reply:
yeah I'm fine switching to `npm`, just liked the old wording a bit more